client/recordUtil.js

```diff
@@ -6,7 +6,8 @@ const serializer = require('../lib/serializer')
 const valueEquals = require('../lib/valueEquals')
 
 // ['0', '1', '2']
-module.exports.CATEGORY_IDS = Object.values(proto.categories)
+// webkit on iOS <= 10.2 does not support Object.values
+module.exports.CATEGORY_IDS = Object.keys(proto.categories).map((key) => proto.categories[key])
```

Comment:
it would be nice to make this either a global polyfill (`if (!Object.values) {Object.values = ...}`) or split it into a utility function since it is also used elsewhere (ebd7a1bf100ded50d49bf7df5dadda8cad174007)